dev/manual_tests/lib/material_arc.dart

```diff
@@ -406,14 +406,14 @@ class _ArcDemo {
 }
 
 class AnimationDemo extends StatefulWidget {
-  const AnimationDemo({ Key key }) : super(key: key);
+  const AnimationDemo({ Key? key }) : super(key: key);
 
   @override
   State<AnimationDemo> createState() => _AnimationDemoState();
 }
 
 class _AnimationDemoState extends State<AnimationDemo> with TickerProviderStateMixin {
-  List<_ArcDemo> _allDemos;
+  late List<_ArcDemo>? _allDemos;
```

Comment:
Set initial value instead of `late`.
```suggestion
  List<_ArcDemo> _allDemos = <_ArcDemo>[];
```